### ISSUE TITLE
Using nano to create empty file 

### ISSUE THREAD

**Issue by brooksph** [swcarpentry/shell-novice#585]
Under the section "Let’s re-create that file and then move up one directory to /Users/nelle/Desktop/data-shell" you're instructed to use nano to create a file named draft.txt. The next step uses ```ls``` to list the files in the directory. No instructions are provided for exiting nano and because the no text is added to file it is not saved.

**Comment by marwahaha** [comment]
Hmm. So in this section:
![image](https://user-images.githubusercontent.com/2541209/27620348-057857c2-5b97-11e7-9a90-6e13897c5ce0.png)

It wouldn't hurt to add a graphic showing the `nano` screen, and adding some text, and then the commands to quit. Or, we could add a comment after `nano draft.txt` saying (make sure the learners enter text in the file, and close the file with `Ctrl-X`, then `y`, then `Enter`.

**Comment by gdevenyi** [comment]
Hi, contributions of this type are welcome to enhance the lesson.

**Comment by reshama** [comment]
@marwahaha @gdevenyi 
Would it be easier to introduce `touch` command and create a blank file?

**Comment by aliceminotto** [comment]
I tend to agree with @reshamas, I've always used `touch` a lot to create placeholder files I was going to write in later. I think it would also make way easier to show examples or giving some small exercises during the lecture (e.g. moving, renaming and copying files, or even show some wildcard behaviour later on). There's no need to explain about the access and modification time neither in my opinion.

**Comment by scttfrdmn** [comment]
While the suggestion to introduce `touch` appears to have been addressed through #606 the original issue remains unresolved. I just stumbled through this during instructor training the other day. This issue appears twice in the episode. Since the purpose is to regenerate the file again maybe just a quick notation is needed and I propose the following.

In the referenced "Deleting Is Forever" section:

>    ```bash
>    $nano draft.txt
>    ```
>
>Let’s again type in a few lines of text. Once we’re happy with our text, we can press ```Ctrl-O``` to write our data to disk. Once our file is saved, we can use ```Ctrl-X``` to quit the editor and return to the shell.
>
>    ```bash
>    $ls
>    ```

and then later in the "With Great Power Comes Great Responsibility" section:

>    ```bash
>    $mkdir thesis
>    $nano thesis/draft.txt
>    ```
>
>Let’s again type in a few lines of text. Once we’re happy with our text, we can press ```Ctrl-O```  to write our data to disk. Once our file is saved, we can use ```Ctrl-X``` to quit the editor and return to the shell.
>
>    ```bash
>    $ls thesis
>    ```
>

**Comment by colinmorris** [comment]
> My preference would be for a light-touch approach, adding a comment on the same line as the nano command

+1

Re: touch, is it even necessary to re-create `draft.txt` the first time, in the *Removing files and directories* section? We don't interact with it directly at any point. I don't think the instructions we give on deleting a directory change depending on whether it contains any files.

(No strong opinion on how to create the file in the *Moving files and directories* section)
